app/src/main/java/com/example/taskyy/domain/LoginValidator.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.taskyy.domain
+
+interface LoginValidator {
+}
```

Comment:
In my experience, there's not much value in abstracting out business logic. Abstractions only make sense if you have or plan to have at least 2 implementations. But, for business logic I just don't see that happen, even for testing, so this just adds additional boilerplate code to your project.